allennlp/data/tokenizers/pretrained_transformer_tokenizer.py

```diff
@@ -53,6 +55,7 @@ def __init__(
         add_special_tokens: bool = True,
         max_length: Optional[int] = None,
         tokenizer_kwargs: Optional[Dict[str, Any]] = None,
+        custom_dummy_tokens: Optional[Tuple[str, str]] = None,
```

Comment:
How about calling this `verification_tokens` instead?
```suggestion
        verification_tokens: Optional[Tuple[str, str]] = None,
```